scripts/cluster

```diff
@@ -41,7 +41,7 @@ function startServerFromDockerCompose() {
         ;;
       "polybft")
         cd core-contracts && npm install && npm run compile && cd -
-        go run ./consensus/polybft/contractsapi/gen/main.go
+        go run ./consensus/polybft/contractsapi/artifacts-gen/main.go
```

Comment:
Would be very useful if we could create a new command in Makefile with both artifacts and bindings generation.

Such command should look like this (double check if there are some typos):
```make
.PHONY: generate-smart-contract-bindings
generate-smart-contract-bindings:
	go run ./consensus/polybft/contractsapi/artifacts-gen/main.go
	go run ./consensus/polybft/contractsapi/bindings-gen/main.go
```

Also we should update `compile-core-contracts` target as well

```make
.PHONY: compile-core-contracts
compile-core-contracts:
	cd core-contracts && npm install && npm run compile
	$(MAKE) generate-smart-contract-bindings
```

Reply:
I think it's not necessary to compile and gen code on every cluster run, with the Make commands should be enough?

Reply:
> I think it's not necessary to compile and gen code on every cluster run, with the Make commands should be enough?

Contracts are compiled in case polybft is started in a Docker container. It probably makes sense to do so, if we are going to share this script with the end users eventually. 
During development, we are most commonly starting polybft protocol from the binary (and there we don't have smart contract compilation as a step, it is assumed those are already compiled).

Anyhow, my comment here just relates to adding new make command, so that it makes our lives a bit easier (I don't want to bother and memorize which commands I need to run in order to generate artifacts and bindings, therefore it is much easier with make commands approach). Any objections? :slightly_smiling_face:

Reply:
> @Stefan-Ethernal you mean a make command to start the cluster with docker?

My suggestion is to provide a make command for creating both artifacts and bindings at the same time. It is a separate topic whether we need it for starting cluster or not, but it would make our lives easier during development. That's my main goal. :slightly_smiling_face:

Reply:
Done: https://github.com/0xPolygon/polygon-edge/pull/1062/commits/54a4a29ba22aec3c8e3620a6d060448200867977